site/snippets/layouts/prevnext.php

```diff
@@ -0,0 +1,23 @@
+<nav class="mb-24">
+	<?php $columnCount = $page->prevListed(page('docs/guide')->index()->listed()) === null ? 1 : 2; ?>
```

Comment:
I would suggest to move `page('docs/guide')->index()->listed()` outside of the snippet. This would make the snippet more universally usable and we wouldn't call `->index()` three times. So passing it as `$siblings` or so to the snippet.